nx-X11/programs/Xserver/hw/nxagent/Font.c

```diff
@@ -1406,9 +1406,10 @@ Bool nxagentDisconnectAllFonts()
 
 static Bool nxagentGetFontServerPath(char * fontServerPath)
 {
-  char path[256];
+  /* ensure path is not longer that fontServerPath) */
+  char path[sizeof(fontServerPath)];
```

Comment:
Can't do this. `fontServerPath` is a pure pointer. We cannot use `sizeof()` here.

Reply:
you are right, will fix that.